src/main/java/de/teamlapen/vampirism/tileentity/TotemTileEntity.java

```diff
@@ -947,32 +960,32 @@ public AxisAlignedBB getRenderBoundingBox() {
         return INFINITE_EXTENT_AABB;
     }
 
-    private static class VillageAttributes implements IVillageAttributes {
-        private final @Nullable
-        IFaction defendingFaction;
-        private final @Nullable
-        IFaction attackingFaction;
+    public static class CaptureInfo implements ICaptureAttributes {
+        @Nullable
+        private final IFaction defendingFaction;
+        @Nullable
+        private final IFaction attackingFaction;
         private final AxisAlignedBB villageArea;
-        private final TotemTileEntity totem;
         private final BlockPos pos;
+        private final boolean shouldForceTargets;
 
-        private VillageAttributes(TotemTileEntity totem) {
+        private CaptureInfo(TotemTileEntity totem) {
```

Comment:
I think for things like this one usually uses a build/create method rather. But I don't really know why.
There is also absolutely no need to change this, just a note.